src/server/rpc/procedures/star-map/star-map.js

```diff
@@ -24,30 +24,34 @@ module.exports = {
      * @return {String}
      */
     getPath: function() {
-        return '/SDSS';
+        return '/sky-map';
     },
 
     arcHourMinSecToDeg: function(arcHour, arcMin, arcSec) {
         return 360.0 * (1.0/24.0) * ( (1.0/60.0) * ( ( (1.0/60.0) * parseFloat(arcSec)) + parseFloat(arcMin)) + parseFloat(arcHour));
     },
 
-    getImage: function(ra, dec, scale, opt) {
+    getImage: function(right_ascension, declination, arcseconds_per_pixel, options) {
```

Comment:
Do you mind switching this to camelcase for consistency? It might be nice to switch to snake case for rpc arguments but, for now, the convention has been camelcase instead.